views/components/GraphTable/helpers.js

```diff
@@ -0,0 +1,15 @@
+import { TOTAL_LABEL } from '../../../constants'
+import { studyCountsToPercentage } from '../../fe-utils/helpers'
+
+export const graphTableRowDataBySiteUtil = (dataBySite) => {
```

Comment:
```suggestion
export const graphTableRowDataBySite = (dataBySite) => {
```

Reply:
gotcha this doesn't need the Util appended
https://github.com/AMP-SCZ/dpdash/pull/193/commits/325e6635a9f09c0b3ce28c6d2a5ab8bfa1369601